Add note to disable ad-blocker when installing JIRA (#5941)
getsentry/sentry-docs: src/docs/product/integrations/issue-tracking/jira/index.mdx

2. Install the Sentry app through the [Jira marketplace](https://marketplace.atlassian.com/apps/1219432/sentry-for-jira?hosting=cloud&tab=overview).

> 3. If you have an ad-blocker enabled you will not be able to complete step number 4.

Reviewer: ```suggestion
3. If you have an ad-blocker enabled, disable it now or you won't be able to complete step number 4.
```

> 4. Select which Sentry organizations you’d like to use with Jira. A link back to Sentry will be presented.

Reviewer: ```suggestion
4. Select which Sentry organizations you’d like to use with Jira. Then click the link back to [sentry.io](https://sentry.io) that's presented.
```

> 5. In Sentry, you’ll see a new Jira instance appear on the Integrations page.

Reviewer: ```suggestion
5. In [sentry.io](https://sentry.io), you’ll see a new Jira instance appear on the **Integrations** page.
```

2. Install the Sentry app through the [Jira marketplace](https://marketplace.atlassian.com/apps/1219432/sentry-for-jira?hosting=cloud&tab=overview).

3. Select which Sentry organizations you’d like to use with Jira, and **Save Settings**.

>   * If you have an ad-blocker, disabled it now before you try to save the settings
>   * A link back to Sentry will be presented

Reviewer: ```suggestion
  * If you have an ad-blocker, disable it now before you try to save the settings. A link back to Sentry will be presented.
```
